mail/templates/success/body.html

```diff
@@ -0,0 +1,19 @@
+{% extends "email_base.html" %}
+{% load static %}
+
+{% block content %}
+  <tr>
+      <td style="background-color: #ffffff;">
+        <p>Hello, your video is ready!</p>
+
+        <p>Title: <a href={{ video_url }}>{{video_title}}</a></p>
+        <p>Collection Title: <a href={{ collection_url }}>{{collection_title}}</a></p>
```

Comment:
```suggestion
        <p>Video: <a href={{ video_url }}>{{video_title}}</a></p>
        <p>Collection: <a href={{ collection_url }}>{{collection_title}}</a></p>
```

I'm confused by the labels "Title" and "Collection Title" I think "Video" and "Collection" will work better.